tsconfig.json

```diff
@@ -22,6 +22,6 @@
     // the vast majority of browsers are probably above that too already. It's
     // most likely safe to up this value, and it would probably result in a
     // smaller package size.
-    "target": "ES5",
+    "target": "ES6",
```

Comment:
**note:** we can make this ES5 but it would be a pain to rewrite the generators. however, this would be a breaking change